lmms_eval/tasks/video-tt/gpt_utils.py

```
@@ -0,0 +1,269 @@
+import ast
+from decord import VideoReader, cpu
```

Ruff (0.11.9)

`decord.VideoReader` imported but unused

Remove unused import

(F401)